docs/guides/saturn-0/whats-new.mdx

```diff
@@ -7,42 +7,38 @@ serve as an introduction to some new and exciting changes to the Rocket Pool pro
 
 - Nodes with an RPL stake of any size qualify for RPL issuance rewards. This unlocks RPL rewards for Node Operators under the 10% collateral cliff 
 who were previously unqualified for RPL issuance rewards. 
 
 - This means nodes with an RPL stake of 0-15% of borrowed ETH receive the maximum APY on RPL. Nodes still earn RPL yield beyond 15% collateral, at a decreasing rate.
 Feel free to take a look how RPL issuance beyond 15% borrowed ETH is defined in [RPIP-30](https://rpips.rocketpool.net/RPIPs/RPIP-30#specification).
 
-- Saturn 0 minipools have a 5% base ETH comission, which will be temporarily boosted up to 14% after [Reward Tree Spec v10](https://rpips.rocketpool.net/assets/rpip-62/rewards-calculation-spec), up until the launch of Saturn 1.
-
 ## New Commission Structure (Saturn 0 part 2)
 ::: tip NOTE
 Dynamic Commission will be enabled when v10 of the [Reward Tree Spec](https://rpips.rocketpool.net/assets/rpip-62/rewards-calculation-spec) is implemented. It's currently in development, so keep an eye on [#protocol](https://discord.com/channels/405159462932971535/405163979141545995) 
 announcements in our [Discord Server](https://discord.com/invite/rocketpool) for updates!
 :::
 
-We're excited to announce that newly launched Saturn 0 minipools will feature a commission structure ranging from 5% to 14%.
+We're excited to announce that newly launched Saturn 0 minipools will feature a commission structure ranging from 10% to 14%.
 
-- Nodes without any RPL stake will be entitled to a 5% base ETH commission.
-- Nodes with staked RPL recieve up to a 4% boost for staked RPL valued at >= 10% of borrowed ETH.
-- Until Saturn 1, nodes receive a 5% dynamic comission boost.
+- Nodes without any staked RPL will earn 10% commission (5% from the minipool contract, and 5% from the dynamic commission boost).
+- Nodes with staked RPL receive up to an additional 4% dynamic commission boost for staked RPL valued at >= 10% of borrowed ETH.
 
-For a total of up to 14% commission on borrowed ETH. 
+Note that the dynamic commission boost will be reduced to 0% on the 5th reward snapshot after Saturn 1 is released (leaving only the 5% minipool contract commission). This serves to increase the yield advantage of migrating to megapools.
 
 ::: warning WARNING
-Your node must be opted into the [Smoothing Pool](/guides/node/fee-distrib-sp#the-smoothing-pool) to benefit from the dynamic comission boost, but don't worry. The latest Smart Node version `v1.14.0` will inform you 
-to opt into the Smoothing Pool when depositing a minipool if you aren't already opted in. *You new minipools will miss out on additional yield if your node 
+Your node must be opted into the [Smoothing Pool](/guides/node/fee-distrib-sp#the-smoothing-pool) to benefit from the dynamic comission boost. Don't worry, the latest Smart Node version `v1.14.0` will inform you
```

Comment:
Didn't comment on it initially, but the "don't worry" feels a little patronizing to me. Doesn't fit in well with the style the docs are usually written in, in my opinion.

Reply:
Sure, removed it